client-js/app/elements/labrad-registry.html

```diff
@@ -6,40 +6,188 @@
       display: block;
     }
     
-    .list {
+    #reg-area {
       padding-top: 12px;
       background-color: white;
       display: inline-block;
       @apply(--shadow-elevation-2dp);
+    }
+    .iron-selected {
+      background-color: #ddd;
+    } 
+    .iron-selector {
+      width: 100%;
+      display: block;
+      border: 1px solid #ccc;
+      border-top: none;
+    }  
+    .reg-list { 
+      padding-top: 12px;
+      display: table; 
+      border: 1px solid;
+    }
+    .row { 
+      display: table-row; 
+      padding: 12px 24px;
+      width: 100%;
+    }
+
+    .head-item { 
+      display: table-cell; 
+      padding: 12px 24px;
+      border: 1px solid;
+      border-color: yellow;
+      width: 100%;
+    } 
+    .head-grabber { 
+      display: table-cell; 
+      *padding: 12px 24px;
+      border: 1px solid;
+      border-color: yellow;
+    } 
+
+    tbody {
+      display:table-header-group;
     }    
+
+    .table-header {
+      @apply(--paper-font-body1);
+      color: var(--secondary-text-color);
+      *text-align: middle;
+      border-bottom: 1px solid;
+      border-bottom-color: #e5e5e5;
+      display:table-header-group;
+
+    } 
+
+    paper-dialog.size-position {
+      position: fixed;
+      top: 16p
+      right: 16px;
+    }
+    paper-dialog.size-position {
+      width: 300px;
+      height: 300px;
+    }
+    menu-button.done {
+      color: #0F9D58;
+    }
   </style>
   <template>
-  <div class="list short">
-    <table>
-      <thead>
-        <th>Key/Dir Name</th>
-        <th>Key Value</th>
-      </thead>
-      <tbody>
-        <template is="dom-repeat" items="{{dirs}}">
-          <tr>
-            <td>
-              <a is="app-link" path="{{item.url}}" href="{{item.url}}">
-               <dir-item kind="folder">{{item.name}}</dir-item>
-              </a>
-            </td>
-            <td></td>
-          </tr>
-        </template>
-        <template is="dom-repeat" items="{{keys}}">
-          <tr>
-            <td><dir-item kind="key">{{item.name}}</dir-item></td>
-            <td><span class="paper-font-body1">{{item.value}}</span></td>
-          </tr>
-        </template>
-      </tbody>
-    </table>
+    <div id="reg-menu"> 
+    <section onclick="clickHandler(event)">
+      <menu-button data-dialog="doDelete" name="delete" icon-name="delete" class = "done"></menu-button>
+      <menu-button data-dialog="newFolderDialog" name="new folder" icon-name="folder-open" class = "done"></menu-button>
+      <menu-button data-dialog="copyDialog" name="copy" icon-name="content-copy" class = "done"></menu-button>
+      <menu-button data-dialog="renameDialog" name="rename" icon-name="code" class= "done"></menu-button>
+      <menu-button data-dialog="newKeyDialog" name="new key" icon-name="create"  class = "done"></menu-button>
+     </section>
```

Comment:
@joshmutus, I'm guessing you could fix this by adjusting the z-index in css. Unfortunately, I can't get the revision you mention to build on my machine. I think the problem is related to webpack and typescript versions, because I've seen a similar problem on linux that prevents the current master from building on gubuntu if you do a fresh clone. I've made some changes to the build system in #31 that fix all this, so maybe we can hold off on this particular styling issue until you've merged the rest of your change. Then we can get #31 in and go from there.